docs/avatar/index.html

```diff
@@ -76,7 +77,149 @@ <h2 class="heading-3 head-lvl-3" id="available-components">
 						</li>
 					</ul>
 				</aside>
-				<article>avatar</article>
+				<article>
+					<div class="component-description">
+						<h3 class="heading-3 head-lvl-3 stick-top" id="alert">alert</h3>
+
+						<p class="justify">
+							avatars are used to show profile images in headers or profile
+							information sections. They can be in multiple sizes. xs &mdash;
+							for extra small, sm &mdash; for small, md &mdash; for medium, lg
+							&mdash; for large, and xl for &mdash; extra large. They can be
+							round or squircle in shape.
```

Comment:
typo error square

Reply:
or it's a word

Reply:
'squircle' was concious choise. instead of using 'square with rounded corners' as a phrase, I used it.